lib/config_file.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+#config_file.py
+#
+#Combine standard login with option of OAuth2.0
```

Comment:
super nit: there should be a space in between the # and the start of the comment text 

`# like this`

Reply:
Added space